.travis/test.sh

```diff
@@ -40,7 +40,7 @@ if [[ ${TASK} == "if-else" ]]; then
     exit 0
 fi
 
-conda install --yes numpy scipy scikit-learn pandas matplotlib
+conda install --yes numpy nose scipy scikit-learn pandas matplotlib
```

Comment:
why do you install nose?

Reply:
Sklearn's checks require `nose`. Without it new integration test fails.

Reply:
> Sklearn's checks require nose

oh..., I got it.